plugins/linux-pipewire/screencast-portal.c

```diff
@@ -611,6 +611,19 @@ init_screencast_capture(struct screencast_portal_capture *capture)
 
 	blog(LOG_INFO, "PipeWire initialized");
 
+	if (capture->session_handle) {
+		blog(LOG_INFO, "Cleaning previous session %s",
```

Comment:
`LOG_INFO` → `LOG_DEBUG`
Also please add the `[pipewire] ` prefix to the debug message